Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -56,18 +56,21 @@ In order to verify that the messaging endpoint is open as expected, you can surf
 ### 1. Using Cortex XSOAR rerouting
 In this configuration, we will use Cortex XSOAR functionality, which reroutes HTTPS requests that hit the default port (443) to the web server that the integration spins up.
 
-The messaging endpoint needs to be: `<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://my.demisto.live/instance/execute/teams`
+The messaging endpoint needs to be:
+
+For XSOAR version less than 8.0.0: `<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://my.demisto.live/instance/execute/teams`.
+
+For XSOAR version 8.0.0. and greater: `https://ext-<CORTEX-XSOAR-Domain>/xsoar/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://ext-my.demisto.live/xsoar/instance/execute/teams`.
 
 The integration instance name, `teams` in this example, needs to be configured in the [Configure Microsoft Teams on Cortex XSOAR](#configure-microsoft-teams-on-cortex-xsoar) step.
 
 The port to be configured in [Configure Microsoft Teams on Cortex XSOAR](#configure-microsoft-teams-on-cortex-xsoar) step should be any available port that is not used by another service.
 
-In addition, make sure ***Instance execute external*** is enabled.
+In addition, make sure ***Instance execute external*** is enabled, (for xsoar version less than 8.0.0).
```

Comment:
```suggestion
In addition, make sure ***Instance execute external*** is enabled (for Cortex XSOAR 6.x).
```
What is ***Instance execute external**?  Is it a command?  a parameter?  an argument?  I could not find it in the yml file.